src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2573,7 +2573,10 @@ public static void GetSystemTimeZones_AllTimeZonesHaveOffsetInValidRange()
             // 0x3E, 0x2C, 0x30, 0x2F, 0x30, 0x2C, 0x4A, 0x33, 0x36, 0x35, 0x2F, 0x32, 0x35, 0x0A
         };
 
-        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        // https://github.com/dotnet/runtime/issues/73031 is the tracking issue to investigate the test failure on Android.
+        private static bool CanRunNJulianRuleTest => PlatformDetection.IsNotAndroid && RemoteExecutor.IsSupported;
```

Comment:
Maybe specifying the platform this way would help disabling?

```suggestion
        private static bool CanRunNJulianRuleTest => !PlatformDetection.IsLinuxBionic && RemoteExecutor.IsSupported;
```

Reply:
It is weird because the failure showing `windows.10.amd64.android.open` which I assume it is `Android` too and the test shouldn't run. 

@MattGal does `windows.10.amd64.android.open` is scoped to Android only too? if so, then the check `PlatformDetection.IsNotAndroid` is not doing the right thing. I'll apply your suggestion @ilonatommy to see if this is going to help.

Reply:
> It is weird because the failure showing `windows.10.amd64.android.open` which I assume it is `Android` too and the test shouldn't run.
> 
> @MattGal does `windows.10.amd64.android.open` is scoped to Android only too? if so, then the check `PlatformDetection.IsNotAndroid` is not doing the right thing. I'll apply your suggestion @ilonatommy to see if this is going to help.

Yes, generally if a queue has "android" in the name it's going to be meant for running tests on Android devices, either emulators in the case of the ubuntu ones or physical devices in the case of the windows queues (I don't fully remember why these are windows machines other than it makes troubleshooting a little easier when the devices go awry).